src/components/BookCard/BookCard.css

```diff
@@ -39,6 +41,27 @@
   color: var(--title-color);
 }
 .card-author {
+	font-size: medium;
+	font-weight: 500;
+	color: var(--author-color);
+	margin-bottom: 1em;
+	font-style: italic;
+}
+
+.dark .card {
+	display: flex;
```

Comment:
I don't think we should be overwriting this many stypes in the dark version of a component (especially something like display). Remember that `.card` also provides some styles to the component that aren't lost, just overwritten by those here. Things like layout, text format/size, etc. should probably be the same light or dark. Things like color, border, etc. should be the things overwritten by these styles.